rdt/transformers/id.py

```diff
@@ -167,7 +203,7 @@ def _warn_not_enough_unique_values(self, sample_size):
         """
         warned = False
         if sample_size > self.generator_size:
-            if self.enforce_uniqueness:
+            if self.cardinality_rule == 'unique':
```

Comment:
Since this can only be `self.cardinality_rule == 'unique'` or `self.cardinality_rule -> None` this can be skipped and just use `self.cardinality_rule`.
```
>>> bool('unique')
True
>>> bool(None)
False
>>> 
```

Reply:
I think we can keep it the way it is since we are going to add other options after https://github.com/sdv-dev/RDT/issues/950